### ISSUE TITLE
Just doesn't work

### ISSUE THREAD

**Issue by marcostux06** [snuffyDev/Beatbump#121]
First time I try this platform and I think it is very good, I hope it can work. 

![2022-11-04 01_39_37-What I've Done - Beatbump - Brave](https://user-images.githubusercontent.com/55907119/199888915-7c2abe2c-e563-402c-830a-1ba1abca4cb8.png)

**Comment by IronMaltese** [comment]
It looks like you're using a much older version of Beatbump, make sure you're on

Https://Beatbump.ml

**Comment by IronMaltese** [comment]
Can you play music on any page?

**Comment by marcostux06** [comment]
> 

Yes, will it affect that I use Brave browser? should I make any changes in browser settings?

**Comment by marcostux06** [comment]
Oh, another error appears in Firefox
![2022-11-04 15_36_57-What I've Done - Beatbump — Mozilla Firefox](https://user-images.githubusercontent.com/55907119/200050587-f1d68e8f-5da5-4ca5-8c2d-328f1012a668.png)

**Comment by IronMaltese** [comment]
> Yes, will it affect that I use Brave browser? should I make any changes in browser settings?

I also use Brave, so this isn't necessary

**Comment by marcostux06** [comment]
> > Yes, will it affect that I use Brave browser? should I make any changes in browser settings?
> 
> I also use Brave, so this isn't necessary

So why is it failing? I hope they fix it, I thought it was an extraordinary platform.

**Comment by IronMaltese** [comment]
I'm not a developer, but development is still active, this issue will be updated once it's fixed

**Comment by martadinata666** [comment]
As i myself cant replicate it i kind sus about region thing, there are  reports about unable to play due geoblocking. Another one is browser extension that blocking `js` kind. But cant say for sure.

**Comment by marcostux06** [comment]
> As i myself cant replicate it, i kinda sus about region thing, there are reports about unable to play due geoblocking. Another one is browser extension that blocking `js` kind. But cant say for sure.

I tried with a VPN and it works! so it's the region, can it be solved? thanks for your answer.

**Comment by IronMaltese** [comment]
Try enabling HlS instead of Http in settings without a vpn and see if it works

**Comment by IntoTheVoid-900** [comment]
This sometimes happens to me, then if I restart my browser a few times it starts working again with some songs, then breaks again, then the whole thing works perfectly.

**Comment by snuffyDev** [comment]
It is due to geo restrictions, I am working on docs, but I will post a guide and reply here with info on how to host the proxy. If you host the proxy in the same region as the Beatbump deploy, then it should work better (still can’t be 100% guaranteed)

**Comment by IntoTheVoid-900** [comment]
I know it would be heavy on the server, but could you proxy the music automatically in the website, similar to Indivious? It could be a toggle like Indivious

**Comment by snuffyDev** [comment]
@IntoTheVoid-900 Right now I’m planning out how to refactor the proxy situation to be similar.

Good news is the rust proxy is just about production ready, so it wouldn’t be too taxing, relatively speaking.

**Comment by snuffyDev** [comment]
Update: I'm currently trying to figure out how to go about getting a few VPS's in different regions
